qiskit-superstaq/qiskit_superstaq/superstaq_provider.py

```diff
@@ -470,7 +470,10 @@ def cq_compile(
         )
 
     def supercheq(
-        self, files: List[List[int]], num_qubits: int, depth: int
+        self,
+        files: List[List[int]],
+        num_qubits: int,
+        depth: int,
     ) -> Tuple[List[qiskit.QuantumCircuit], npt.NDArray[np.float_]]:
         """Docstring."""
```

Comment:
lol (predates this pr)
```suggestion
        """Returns the randomly generated circuits and the fidelity matrix for inputted files."""
```